extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1080-lms-assignment.sql

```diff
@@ -48,6 +48,7 @@ BEGIN
 			ON assignmentCatDescriptor.DescriptorId = AssignmentCategoryDescriptor.AssignmentCategoryDescriptorId
 
 
+	-- Insert new records
```

Comment:
While there's no harm to this comment, does it add anything? After all, the next line says `INSERT INTO...` 😉 .

Reply:
Yes, probably we can remove it, I don't know, probably I added it thinking about having consistent commits or something like that 😅